File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/nullability/SpecialThirdPartyNullability.java
// !ADD_JAVA_API
import javaApi.View;

class C {
    void foo(View parentView) {
        View v1 = parentView.findViewById(5);
        if (v1 == null) {
            System.out.println("view1 is null");
        }

        View v2 = (View) parentView.findViewById(5);
        if (v2 == null) {
            System.out.println("view2 is null");
        }
    }
}

File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/nullability/SpecialThirdPartyNullability.kt
import javaApi.View

internal class C {
    fun foo(parentView: View) {
        val v1 = parentView.findViewById<View>(5)
        if (v1 == null) {
            println("view1 is null")
        }

        val v2 = parentView.findViewById(5) as View?
        if (v2 == null) {
            println("view2 is null")
        }
    }
}
